src/database/schema/UserSchema.ts

```diff
@@ -199,7 +208,11 @@ class UserSchema implements BaseDbSchema {
       });
     };
 
-    db.transaction(resolver);
+    try {
+      db.transaction(resolver);
+    } catch (e) {
+      console.log('error resolver', e);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a try-catch block around the `db.transaction(resolver)` call is a good practice for error handling. However, logging the error to the console might not be sufficient in a production environment. Consider adding more robust error handling or logging mechanisms.
```diff
-    db.transaction(resolver);
+    try {
+      db.transaction(resolver);
+    } catch (e) {
+      console.log('error resolver', e);
+    }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->